Test failure is unrelated

```
FAILED tests/components/config/test_automation.py::test_update_automation_config_with_error[automation_config0] - assert {'message': "...ta['action']"} == {'message': "...a['trigger']"}
  Differing items:
  {'message': "Message malformed: required key not provided @ data['action']"} != {'message': "Message malformed: required key not provided @ data['trigger']"}
  Full diff:
  - {'message': "Message malformed: required key not provided @ data['trigger']"}
  ?                                                                    - ^^^^
  + {'message': "Message malformed: required key not provided @ data['action']"}
  ?                                                                   ++  ^^
Error: Process completed with exit code 1.
```

--- a/homeassistant/components/august/manifest.json
+++ b/homeassistant/components/august/manifest.json
@@ -2,7 +2,7 @@
   "domain": "august",
   "name": "August",
   "documentation": "https://www.home-assistant.io/integrations/august",
-  "requirements": ["yalexs==1.2.6", "yalexs_ble==1.12.2"],
+  "requirements": ["yalexs==1.2.6", "yalexs_ble==1.12.3"],
   "codeowners": ["@bdraco"],
   "dhcp": [
     {

--- a/homeassistant/components/yalexs_ble/manifest.json
+++ b/homeassistant/components/yalexs_ble/manifest.json
@@ -3,7 +3,7 @@
   "name": "Yale Access Bluetooth",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/yalexs_ble",
-  "requirements": ["yalexs-ble==1.12.2"],
+  "requirements": ["yalexs-ble==1.12.3"],
   "dependencies": ["bluetooth"],
   "codeowners": ["@bdraco"],
   "bluetooth": [

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -2617,7 +2617,7 @@ yalexs-ble==1.12.3
 yalexs==1.2.6
 
 # homeassistant.components.august
-yalexs_ble==1.12.2
+yalexs_ble==1.12.3
 
 # homeassistant.components.yeelight
 yeelight==0.7.10

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -2611,7 +2611,7 @@ xs1-api-client==3.0.0
 yalesmartalarmclient==0.3.9
 
 # homeassistant.components.yalexs_ble
-yalexs-ble==1.12.2
+yalexs-ble==1.12.3
 
 # homeassistant.components.august
 yalexs==1.2.6

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -1824,7 +1824,7 @@ xmltodict==0.13.0
 yalesmartalarmclient==0.3.9
 
 # homeassistant.components.yalexs_ble
-yalexs-ble==1.12.2
+yalexs-ble==1.12.3
 
 # homeassistant.components.august
 yalexs==1.2.6

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -1830,7 +1830,7 @@ yalexs-ble==1.12.3
 yalexs==1.2.6
 
 # homeassistant.components.august
-yalexs_ble==1.12.2
+yalexs_ble==1.12.3
 
 # homeassistant.components.yeelight
 yeelight==0.7.10

